source/habitat_risk_assessment.rst

```diff
@@ -77,28 +75,26 @@ The risk of human activities to habitats or species is modeled in five steps.
 
 .. math:: E_{jkl} = \frac{\sum^N_{i=1}\frac{e_{ijkl}}{d_{ijkl}\cdot w_{ijkl}}} {\sum^N_{i=1}\frac{1}{d_{ijkl} \cdot w_{ijkl}}}
-   :label: eq1
+   :label: exposure
 
 .. math:: C_{jkl} = \frac{\sum^N_{i=1}\frac{c_{ijkl}}{d_{ijkl}\cdot w_{ijkl}}}{\sum^N_{i=1}\frac{1}{d_{ijkl} \cdot w_{ijkl}}}
-   :label: eq2
+   :label: consequence
 
 where :math:`E_{jkl}` is the exposure score specific to habitat *j*, from stressor *k* in location *l*; :math:`C_{jkl}` is the consequence score, :math:`e_{ijkl}` is the exposure rating criterion *i*, specific to habitat *j* and stressor *k* and location *l*; :math:`c_{ijkl}` is the consequence rating. :math:`d_{ijkl}` represents the data quality rating, :math:`w_{ijkl}` represents the importance weighing for criterion. *N* is the number of criteria evaluated for each habitat.
 
 **Step 2.** The second step combines the exposure and response values to produce a risk value for each stressor-habitat combination in each grid cell. There are two options for risk calculation.
 
-For Euclidean Risk calculation, risk to habitat *j* caused by stressor *k* in each location (i.e. cell) *l* is calculated as the Euclidean distance from the origin in the exposure-consequence space, where average exposure (:ref:`eq1`) is on one axis and the average consequence score (:ref:`eq2`) is on the other.
+For Euclidean Risk calculation, risk to habitat *j* caused by stressor *k* in each location (i.e. cell) *l* is calculated as the Euclidean distance from the origin in the exposure-consequence space, where average exposure (:eq:`exposure`) is on one axis and the average consequence score (:eq:`consequence`) is on the other.
 
 .. math:: R_{jkl} = \sqrt{(E_{jkl}-1)^2+(C_{jkl}-1)^2}
-   :label: eq3
```

Comment:
Is there a reason for removing the labels here? I'm just wondering if they might be referenced in the HRA source code? I believe it is something we've done in the past, though that kind of coupling is probably not ideal given the need to upkeep both.

Reply:
Having multiple `eq1`s in different model files raises a warning. Each label is supposed to be unique. Giving them more descriptive labels is kinda tedious so I just deleted the ones that aren't referenced anywhere. And the labels that are actually displayed seem to be  different. E.g. the equation labeled `:label: cbc_emissions` in https://storage.googleapis.com/releases.naturalcapitalproject.org/invest-userguide/latest/coastal_blue_carbon.html#the-model shows up with a label (5) [here](https://storage.googleapis.com/releases.naturalcapitalproject.org/invest-userguide/latest/coastal_blue_carbon.html#the-model). I think they always are numbered from the top of the page.

Reply:
Oh, fun! I didn't know that, thanks.